docs/website-v2/versioned_docs/version-1.0.0/lte/config_agw.md

```diff
@@ -93,13 +93,13 @@ MHYwEAYHKoZIzj0CAQYFK4EEACIDYgAECMB9zEbAlLDQLq1K8tgCLO8Kie5IloU4QuAXEjtR19jt0KTk
 Navigate to "Gateways" on the NMS via the left navigation bar, hit
 "Add Gateway", and fill out the form using the hardware secrets from above:
 
-![Creating a gateway](assets/nms/create_agw.png)
+![Creating a gateway](pathname:///assets/nms/create_agw.png)
```

Comment:
Another page with broken image urls. Ignored by using `pathname://` prefix, but you should rather fix those URLs anyway